dietpi/dietpi-software

```diff
@@ -1723,6 +1723,13 @@ INDEX_BROWSER_TARGET=$INDEX_BROWSER_TARGET"
 		aSOFTWARE_DESC[$software_id]='file manager with vi bindings'
 		aSOFTWARE_CATX[$software_id]=22
 
+		software_id=198
+
+		aSOFTWARE_NAME[$software_id]='File Browser'
+		aSOFTWARE_DESC[$software_id]='web based file manager'
+		aSOFTWARE_CATX[$software_id]=22
```

Comment:
While it is sorts of a file manager, actually I would add this to the cloud category, where Syncthing, Nextcloud ownCloud etc reside. MC and ViFM are GUI file managers without and network capabilities.

The question is where most users would search or expect it.

Reply:
Well `filebrowser` is nothing else than a MC with web interface instead of GUI 😉 
Anyway I guess it doesn't matter really. Will change it into category 4

Reply:
A large difference is that it cannot really be used to share files with others, as long as you do not create a new user with overlapping data directory or so. But it does make your files available and downloadable remotely and is hence a cloud software as well. Yeah not sure, one benefit of leaving it in the file manager category is that it is pretty empty. For docs on the other hand this means to add new ones for the other two as well to not have File Browser alone there. And actually both are such install options which I rather want to remove, as its not more than `apt install` 😄.

Reply:
changed it into category 4